module.json

```diff
@@ -7,7 +7,8 @@
   "homepage": "https://github.com/ARMmbed/mbed-client",
   "license": "Apache-2.0",
   "dependencies": {
-    "mbed-client-c": "^2.0.0"
+    "mbed-client-c": "^2.0.0",
+    "mbed-trace": "ARMmbed/mbed-trace"
```

Comment:
The version number is currently 0.2.1 so I think I have to change that to something like: